scripts/serverless.js

```diff
@@ -140,6 +140,12 @@ processSpanPromise = (async () => {
       const args = process.argv.slice(2);
       const firstParamIndex = args.findIndex(isParamName);
       const commands = args.slice(0, firstParamIndex === -1 ? Infinity : firstParamIndex);
+
+      if (commands.join('') === 'dev') {
+        process.argv[2] = '--dev';
```

Comment:
Let's also update the comment above, that it's also about `serverless dev` to `serverless --dev`